boltdesignsystem/bolt
# Hotfix: UIKit Fixes for IE 11 / Polyfills

## Jira
http://vjira2:8080/browse/BDS-1883

## Summary
Updates the logo styles + cross browser polyfills / Babel transpiling used for Pattern Lab's UI to address recent UI issues flagged by @danielamorse in IE 11 (Windows 7).

### Before
![image](https://user-images.githubusercontent.com/1617209/67865935-8adeb880-fafe-11e9-98b3-59a99b04d478.png)

### After
![image](https://user-images.githubusercontent.com/1617209/67865882-74d0f800-fafe-11e9-99e5-6c5c096fe2d0.png)


## Details
- Moves the UIKit polyfills into their own standalone JS package to make it easier to manage this very specific set of dependencies
- Updates the two Pattern Lab JS entrypoints to pull in and use this new `@bolt/uikit-polyfills` package
- Minor UIKit-specific Webpack build adjustments
- Minor Logo-specific styles updates

> Side note: the upcoming Bolt pollyfill updates will probably be pretty close to this!

## How to test
- Review changes
- Confirm Pattern Lab's UI is once again working as expected in IE 11, Chrome, etc, including internal nav links work as expected (ie. not open in a new window)

CC @bmuenzenmeyer — while I don't think this is a recent regression, there's a good chance I'll be porting these updates back into PL Node in the near future!
